opentreemap/treemap/udf.py

```diff
@@ -630,10 +674,7 @@ def _validate_single_datatype(self, datatype):
             raise ValidationError(_('invalid datatype'))
 
         if datatype['type'] in ('choice', 'multichoice'):
-            choices = datatype.get('choices', None)
-
-            if choices is None:
-                raise ValidationError(_('Missing choices key'))
```

Comment:
I removed this validation error because it's now possible (and valid) for the "choices" key to be missing. For example, if you define "protected_choices" instead, you don't need "choices" (Ex. https://github.com/OpenTreeMap/otm-core/blob/develop/opentreemap/works_management/models.py#L68). Also, there is another validation constraint which checks that there is at least 1 choice defined, which I think is sufficient. No unit tests failed as a result of this change.